src/CalendarAvailability.vue

```diff
@@ -316,5 +315,8 @@ export default {
 	max-width: 500px;
 	width: 100%;
 	gap: 24px;
+	border-top: 1px solid var(--color-border);
+	padding-top: 10px;
+	margin-top: 5px;
```

Comment:
Can you use `calc()` based on `var(--default-grid-baseline)` here? :) We want to move away from arbitrary values towards this based on the baseline (which is 4px).

Reply:
Not sure to understand what you exactly expect @jancborchardt ; is that this ?

```
margin-top: calc(var(--default-grid-baseline));
margin-bottom: calc(var(--default-grid-baseline));
```

Reply:
Depends how much you would like – but if you only want the base value, you don't need calc() :)

So e.g. you can use:

```
margin-top: calc(2 * var(--default-grid-baseline));
margin-bottom: var(--default-grid-baseline);
```

That would come out at 8px top and 4px bottom. Depending on how it looks good and symmetric. :)

Reply:
OK, understood ; check my last commit @jancborchardt